instrumentation/cassandra/cassandra-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/cassandra/v3_0/CassandraNetworkAttributesGetter.java

```diff
@@ -7,11 +7,13 @@
 
 import com.datastax.driver.core.ExecutionInfo;
 import io.opentelemetry.instrumentation.api.semconv.network.NetworkAttributesGetter;
+import io.opentelemetry.instrumentation.api.semconv.network.ServerAttributesGetter;
 import java.net.InetSocketAddress;
 import javax.annotation.Nullable;
 
 final class CassandraNetworkAttributesGetter
-    implements NetworkAttributesGetter<CassandraRequest, ExecutionInfo> {
+    implements ServerAttributesGetter<CassandraRequest>,
```

Comment:
adding the interface here isn't useful if you are not going to implement any of the methods and not use it with `ServerAttributesExtractor`

Reply:
this PR https://github.com/open-telemetry/opentelemetry-java-instrumentation/commit/9cb157492c366b09ad06e726dce2f225a2cfa939#diff-9921147bb0390248e2779c790b1666bcfe76df343d4d743577c946ad42a0fadc

Reply:
removed.